src/templates/ssr/fabric-custom-xl/test.json

```diff
@@ -0,0 +1,8 @@
+{
+	"DapAssetsFolder": "GooglePixel2",
+	"ThirdPartyTag": "https://tpc.googlesyndication.com/pagead/imgad?id=CICAgKCrtKGjoAEQARgBMgjYagW3ARF6Fw",
+	"TrackingPixel": "",
+	"ResearchPixel": "",
+	"ViewabilityTracker": "<img src=\"\">",
+	"thirdPartyJSTracking": "<SCRIPT TYPE='application/javascript' SRC='https://pixel.adsafeprotected.com/rjss/st/726370/54949606/skeleton.js'></SCRIPT> <NOSCRIPT><IMG SRC='https://pixel.adsafeprotected.com/rfw/st/726370/54949605/skeleton.gif?gdpr=${GDPR}&gdpr_consent=${GDPR_CONSENT_278}&gdpr_pd=${GDPR_PD}' BORDER=0 WIDTH=1 HEIGHT=1 ALT=''></NOSCRIPT>"
+}
```

Comment:
Is this just the same test data as the Fabric Custom? I think it would be preferable to use data that's specific to the XL size (maybe ask Scott?). Otherwise this local preview is just exactly the same as the Fabric Custom, and doesn't check that the XL size behaves as we expect. Would also make any future visual regression testing of this template more meaningful

Reply:
Great idea, I've updated the test data by updating the `DapAssetsFolder` to `DAP_ad_awards_XL`, Ive also updated the screenshot above in the description.